tensorflow/core/kernels/relu_op.cc

```diff
@@ -188,6 +191,7 @@ TF_CALL_GPU_NUMBER_TYPES(DECLARE_GPU_SPEC);
 TF_CALL_GPU_NUMBER_TYPES(REGISTER_GPU_KERNELS);
 #undef REGISTER_GPU_KERNELS
 
+#if GOOGLE_CUDA
```

Comment:
Doesn't this disable the CPU implementation?

Reply:
It does not seem so.

This region of code (lines 77 - 217) is within the `#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM` block

So this `#if GOOGLE_CUDA` will disable only the ROCm implementation.

Reply:
Yeah, your are right. Thanks for the explanation.